clmm/galaxycluster.py

```diff
@@ -27,45 +20,63 @@ class GalaxyCluster():
     galcat : astropy Table
         Table of background galaxy data containing at least galaxy_id, ra, dec, e1, e2, z
     """
-    def __init__(self, unique_id: str, ra: float, dec: float, z: float,
+    def __init__(self, *args, **kargs):
```

Comment:
I think `kargs` is generally written as `kwargs`?

Reply:
Yeah, it was a typo on my side :expressionless: